src/ralph/defaults.py

```diff
@@ -35,9 +38,67 @@ class StorageBackends(Enum):
     FS = "ralph.backends.storage.fs.FSStorage"
 
 
+def load_config():
+    """Return a dictionnary representing Ralph's configuration"""
+
+    config_path = APP_DIR / "config.yml"
```

Comment:
We better shouldn't hard code the configuration path (see also next suggestion):

```suggestion
def load_config(config_file_path):
    """Return a dictionnary representing Ralph's configuration"""

```

Reply:
Fixed